deepchem/rl/a3c.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, value_weight, entropy_weight, **kwargs):
 
   def create_tensor(self, **kwargs):
     reward, action, prob, value = [layer.out_tensor for layer in self.in_layers]
-    log_prob = tf.log(prob)
+    log_prob = tf.log(prob + 0.0001)
```

Comment:
I added np.finfo(np.float32).eps.  I'm willing to take the hit into the distortion of probability distribution for the added simplicity.